conseil-api/src/main/scala/tech/cryptonomic/conseil/api/ConseilOperations.scala

```diff
@@ -0,0 +1,104 @@
+package tech.cryptonomic.conseil.api
+
+import com.github.ghik.silencer.silent
+import slick.jdbc.PostgresProfile.api._
+import tech.cryptonomic.conseil.common.tezos.SqlOperations.{AccountResult, BlockResult, OperationGroupResult}
+import tech.cryptonomic.conseil.common.tezos.TezosTypes.{AccountId, BlockHash}
+import tech.cryptonomic.conseil.common.tezos.{SqlOperations, Tables, TezosDatabaseOperations => TezosDb}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class ConseilOperations extends SqlOperations {
```

Comment:
We'd need to add comments here for completeness, even a reminder will do for now, for our team to fill later

Reply:
Added similar to `SqlOperations`, if you find that incorrect, please update that later :)